feedback/edxapp_wrapper/student_models.py

```diff
@@ -0,0 +1,18 @@
+"""
+Courseware generalized definitions.
+"""
```

Comment:
^ Same comment here. We understand this kind of architecture, but others do not necessarily. So we're adding extra complexity. What pattern is used in this repos for the Open edX org?